test/test_json_util.py

```diff
@@ -92,61 +92,52 @@ def test_dbref(self):
-        # No explicit offset
-        jsn = '{"dt": { "$date" : "1970-01-01T00:00:00.000"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1970-01-01T00:00:00"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1970-01-01T00:00:00.000000"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        # Localtime behind UTC
-        jsn = '{"dt": { "$date" : "1969-12-31T16:00:00.000-0800"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1969-12-31T16:00:00.000000-0800"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1969-12-31T16:00:00.000-08:00"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1969-12-31T16:00:00.000000-08:00"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1969-12-31T16:00:00.000000-08"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        # Localtime ahead of UTC
-        jsn = '{"dt": { "$date" : "1970-01-01T01:00:00.000+0100"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1970-01-01T01:00:00.000000+0100"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1970-01-01T01:00:00.000+01:00"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1970-01-01T01:00:00.000000+01:00"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
-        jsn = '{"dt": { "$date" : "1970-01-01T01:00:00.000000+01"}}'
-        self.assertEqual(EPOCH_AWARE, json_util.loads(jsn)["dt"])
+                                                    49, 45, 191000, utc)}, json_options=tz_aware_opts)
+        self.round_trip({"date": datetime.datetime(2009, 12, 9, 15,
+                                                   49, 45, 191000)})
+
+        for jsn in ['{"dt": { "$date" : "1970-01-01T00:00:00.000+0000"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00.000000+0000"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00.000+00:00"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00.000000+00:00"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00.000000+00"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00.000Z"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00.000000Z"}}',
+                    '{"dt": { "$date" : "1970-01-01T00:00:00Z"}}']:
+            self.assertEqual(EPOCH_AWARE, json_util.loads(
+                jsn, json_options=tz_aware_opts)["dt"])
+            self.assertEqual(EPOCH_NAIVE, json_util.loads(jsn)["dt"])
+
+
+        for jsn in ['{"dt": { "$date" : "1970-01-01T00:00:00.000"}}',
```

Comment:
Can this `for` be combined with the above `for`? It looks like the body is the same in both cases.

Reply:
Done.